scripts/apidoc/utils.ts

```diff
@@ -105,11 +106,28 @@ export function extractRawExamples(signature?: SignatureReflection): string[] {
   );
 }
 
+/**
+ * Extracts the all see also references from the jsdocs separately.
```

Comment:
This sentence doesn't seem right to me. Did you wanted to write this?
```suggestion
 * Extracts all the see also references from the jsdocs separately.
```

But I don't know why the "also" is needed (for the function name as well).

Reply:
```suggestion
 * Extracts all the `@see` references from the jsdocs separately.
```